README.rst

```diff
@@ -68,29 +70,29 @@ Pre-requisites:
 
 * On Windows, please follow the `Comprehensive Installation instructions
   <https://github.com/nexB/scancode-toolkit/wiki/Comprehensive-Installation>`_.
-  Make sure you use Python 2.7 32 bits from
-  https://www.python.org/ftp/python/2.7.15/python-2.7.15.msi
+  Make sure you use Python 3.6.8 32 bits from
+  https://www.python.org/ftp/python/3.6.8/python-3.6.8.exe
 
-* On macOS, install Python 2.7 from
-  https://www.python.org/ftp/python/2.7.15/python-2.7.15-macosx10.6.pkg
+* On macOS, install Python 3.6.8 from
+  https://www.python.org/ftp/python/3.6.8/python-3.6.8-macosx10.6.pkg
 
   Next, download and extract the latest ScanCode release from
   https://github.com/nexB/scancode-toolkit/releases/
 
-* On Linux install the Python 2.7 "devel" and these packages using your
+* On Linux install the Python 3.6 "devel" and these packages using your
   distribution package manager:
 
   * On Ubuntu 14, 16 and 18 use:
-    ``sudo apt-get install python-dev xz-utils zlib1g libxml2-dev libxslt1-dev bzip2``
+    ``sudo apt-get install python3.6-dev xz-utils zlib1g libxml2-dev libxslt1-dev bzip2``
```

Comment:
Ubuntu 14 and 16 do not have Python 3.6 in directly asvailible from `apt-get`. But it can be supported with [installing from PPA](https://askubuntu.com/a/865569).

Reply:
:+1:

Reply:
I added notice about PPA.